Package.swift

```diff
@@ -17,5 +20,19 @@
 import PackageDescription
 
 let package = Package(
-    name: "Kitura-TemplateEngine"
+    name: "Kitura-TemplateEngine",
+    products: [
+        // Products define the executables and libraries produced by a package, and make them visible to other packages.
```

Comment:
I think we can delete this line - it's just boilerplate generated by SPM when you do `swift package init`.